src/_guides/language/effective-dart/style.md

```diff
@@ -174,7 +174,7 @@ class Dice {
 <?code-excerpt "misc/lib/effective_dart/style_bad.dart (const-names)"?>
 {% prettify dart %}
 const PI = 3.14;
-const kDefaultTimeout = 1000;
```

Comment:
This is meant to be a _bad_ example.

Reply:
> Is the k in the BAD example because Effective Dart used to say something about prefixes?

Oops, missed this comment: yes I believe so, even if we don't mention it in the prose. But I'll let Bob confirm.

Reply:
The intent was to show that you should not use any sort of prefix to name a constant. How about leaving this example and also adding a separate guideline at the top:

```
### DON'T use Hungarian notation or any other prefixed letters on identifiers to identify their type or scope.

The type system is more than happy to tell you what the scope, mutability, and type of any identifier is, so there's no need to decorate its name yourself with that information.
```

Reply:
SGTM, though it might make sense to end the **Identifiers** section with that guideline (so we don't have to include good and bad examples. @kwalrath WDYT?

Reply:
@munificent @chalin if the intent was to show that a prefix is not necessary, it definitely doesn't belong to that section. Also, it's pretty unclear what that prefix means, so as a reader, I can say that this intent may not be understood. Also I don't know what Hungarian notation is, so better add an example if you want to include a separate guideline :)
And finally, I still feel like the `lowerCamelCase` section should include `UpperCamelCase` as a bad example, therefore `DefaultTimeout` would make sense as to why it's bad.

Reply:
> Also, it's pretty unclear what that prefix means

The Chrome team has a style (for their C++ code) of using a "k" prefix for constants, and that has bled into some Dart code primarily from Dart developers who used to write C++ in Chrome.

> Also I don't know what Hungarian notation is

https://en.wikipedia.org/wiki/Hungarian_notation

> I still feel like the lowerCamelCase section should include UpperCamelCase as a bad example, therefore DefaultTimeout would make sense as to why it's bad.

+1.